test/tests/rackhd11/test_rackhd11_api_workflows.py

```diff
@@ -89,5 +89,83 @@ def test_api_11_workflows_put_ipmi(self):
         api_data = fit_common.rackhdapi("/api/1.1/workflows/library/" + 'Graph.Obm.Ipmi.CreateSettings.Test')
         self.assertEqual(api_data['status'], 200, 'Incorrect HTTP return code, expected 200, got:' + str(api_data['status']))
 
+    def test_api_11_workflows_waitOn_anyOf(self):
+        data_payload = {
+            "friendlyName": "Multi-REST-call",
+            "injectableName": "Graph.REST.call",
+            "tasks":[
+                {
+                    "label" : "REST-call-good",
+                    "taskDefinition": {
+                        "friendlyName":"good REST call",
+                        "injectableName":"Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/1.1/nodes",
+                            "method":"GET"
+                        },
+                        "properties": {}       
+                    }
+                },
+                {
+                    "label" : "REST-call-2",
+                    "taskDefinition": {
+                        "friendlyName":"2nd REST call",
+                        "injectableName":"Task.Rest.GET",
+                        "implementsTask": "Task.Base.Rest",
+                        "options":{
+                            "url":"http://localhost:8080/api/1.1/nodes",
```

Comment:
The test case may not run in the RackHD host server, so use the `localhost` here is not correct.